### PR TITLE
PD-8833 TravisCI file for cassandra_utils

### PR DESCRIPTION
**Pull request by rvanburen-r7**
#TravisCI update for Cassandra_Utils

Need to check if this repo has TravisCI enabled.

### CODE REVIEW

**Comment by camci-r7** [comment]
${CASSANDRA_UTILS_TEST_RESULTS}

**Comment by jpooler-r7** [comment]
Probably should also kill the jenkins stuff while you're at it, lol.

**Comment by rvanburen-r7** [comment]
Link to PR to remove Jenkins stuff
https://github.com/rapid7/jobs-razorci-library/pull/581

File: .travis.yml [hunk 1]
```diff
@@ -2,4 +2,9 @@ sudo: false
 language: ruby
 rvm:
   - 2.2.4
+sudo: false
```
Comment (fmitchell-r7) [inline .travis.yml]: It looks like there's already a `sudo: false` at line 1, so I don't think you need this one. See the [online validator](https://lint.travis-ci.org/) to check.

**Pull request merged**